lib/ios_platform_images.dart

```diff
@@ -34,34 +39,52 @@ class FutureMemoryImage extends ImageProvider<FutureMemoryImage> {
 
   Future<ui.Codec> _loadAsync(FutureMemoryImage key, DecoderCallback decode) async {
     assert(key == this);
-    return futureBytes.then((Uint8List bytes) {
+    return _futureBytes.then((Uint8List bytes) {
       return decode(bytes);
     });
   }
 
+  /// See [ImageProvider.operator==].
   @override
   bool operator ==(dynamic other) {
     if (other.runtimeType != runtimeType) return false;
     final FutureMemoryImage typedOther = other;
-    return futureBytes == typedOther.futureBytes && scale == typedOther.scale;
+    return _futureBytes == typedOther._futureBytes && scale == typedOther.scale;
   }
 
+  /// See [ImageProvider.hashCode].
   @override
-  int get hashCode => hashValues(futureBytes.hashCode, scale);
+  int get hashCode => hashValues(_futureBytes.hashCode, scale);
 
+  /// See [ImageProvider.toString].
   @override
   String toString() =>
-      '$runtimeType(${describeIdentity(futureBytes)}, scale: $scale)';
+      '$runtimeType(${describeIdentity(_futureBytes)}, scale: $scale)';
 }
 
+/// Class to help loading of iOS platform images into Flutter.
+///
+/// For example, loading an image that is in `Assets.xcassts`.
 class IosPlatformImages {
   static const MethodChannel _channel =
       const MethodChannel('plugins.flutter.io/ios_platform_images');
 
+  /// Loads an image from asset catalogs.  The equivalent would be:
+  /// `[UIImage imageNamed:name]`.
+  ///
+  /// Throws an exception if the image can't be found.
+  ///
+  /// See [https://developer.apple.com/documentation/uikit/uiimage/1624146-imagenamed?language=objc]
   static FutureMemoryImage load(String name) {
     return FutureMemoryImage(_channel.invokeMethod('loadImage', name));
   }
 
+  /// Loads an URL for a resource.  The equivalent would be:
+  /// `[[NSBundle mainBundle] URLForResource:name withExtension:ext]`.
+  ///
+  /// Returns null if the resource can't be found.
+  ///
+  /// See [https://developer.apple.com/documentation/foundation/nsbundle/1411540-urlforresource?language=objc]
   static Future<String> loadURL(String name, [String ext]) {
```

Comment:
I didn't follow up on the comment on the previous PR. It's ambiguous whether it's to find the url given a name or loading the contents of a url which is represented by the name (at a first glance). Maybe resolveUrl?

Reply:
Good call, i like `resolveURL`.  I'll do that as a separate commit.